components/forms/ltft/LtftSummary.tsx

```diff
@@ -45,43 +46,51 @@ const LtftSummary = ({
   const columnHelper = createColumnHelper<LtftSummaryObj>();
 
   // Header
-  const renderNameHeader = ({ column }) => (
+  const renderNameHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
       title="Name"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderCreatedHeader = ({ column }) => (
+  const renderCreatedHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Created date"
+      title="Created"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderStatusHeader = ({ column }) => (
+  const renderLastModifiedHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Status"
+      title="Last modified"
       data-cy={`table-column_${column.id}`}
     />
   );
-  const renderLastModifiedHeader = ({ column }) => (
+  const renderStatusHeader = ({
+    column
+  }: HeaderContext<LtftSummaryObj, string>) => (
     <TableColumnHeader
       column={column}
-      title="Status date"
+      title="Status"
       data-cy={`table-column_${column.id}`}
     />
   );
 
   //Header value
-  const renderValue = props => <span>{props.renderValue()}</span>;
-  const renderDayValue = props => (
+  const renderValue = (props: any) => <span>{props.renderValue()}</span>;
+  const renderDayValue = (props: any) => (
     <span>{dayjs(props.renderValue()).toString()}</span>
   );
 
   // Operation Column
-  const renderOperationColumnValue = props => (
+  const renderOperationColumnValue = (props: any) => (
```

Comment:
and this should work too:
` const renderOperationColumnValue = (props: {
    row: { original: LtftSummaryObj };
  })`...